WordPress/WordPressTest/ReaderStreamViewControllerTests.swift

```diff
@@ -2,13 +2,7 @@ import UIKit
 import XCTest
 @testable import WordPress
 
-class ReaderStreamViewControllerTests: XCTestCase {
-    private var contextManager: ContextManagerMock!
-
-    override func setUp() {
-        contextManager = ContextManagerMock()
-    }
-
+class ReaderStreamViewControllerTests: CoreDataTestCase {
     // Tests that a ReaderStreamViewController is returned
     func testControllerWithTopic() {
         let context = contextManager.mainContext
```

Comment:
Nitpick. Now that we inherit from `CoreDataTestCase`, this could become:

```suggestion
        let context = mainContext
```

Right?